roles/benchmarking_run_nvidiadl_ssd/tasks/main.yml

```diff
@@ -45,33 +45,35 @@
   command:
     oc create -f "{{ artifact_extra_logs_dir }}/001_pod_run-nvidiadl-ssd.yml"
 
-- name: Wait for the benchmark completion
```

Comment:
a bit above here (line 25), could you remove `-n {{ benchmarking_namespace }}` from `oc create` and move it below to `oc apply`?

will make it easier to reuse the files, as in 
```
oc apply -f https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/316/pull-ci-openshift-psap-ci-artifacts-master-test-commit/1488755158606155776/artifacts/test-commit/test-commit/artifacts/recipes_run_cocodataset_nvidiadl_ssd/012__benchmarking__run_nvidiadl_ssd/001_pod_run-nvidiadl-ssd.yml
```
currently the namespace must be `default`

and same idea for https://github.com/Maxusmusti/ci-artifacts/blob/cocobench/roles/benchmarking_run_nvidiadl_ssd/templates/nvidiadl-ssd-pod.yml.j2

if you remove `namespace: {{ benchmarking_namespace }}` from the file, and only define the namespace in `oc create -f "{{ artifact_extra_logs_dir }}/001_pod_run-nvidiadl-ssd.yml" -n "{{ benchmarking_namespace }}"`